econml/inference/_inference.py

```diff
@@ -425,9 +431,12 @@ def effect_inference(self, X, *, T0, T1):
         pred = pred[np.arange(T0.shape[0]), ..., ind]
         pred_stderr = pred_stderr[np.arange(T0.shape[0]), ..., ind] if pred_stderr is not None else None
         # d_t=1 here since we measure the effect across all Ts
+        # Omit treatment name argument to fall back to the default
```

Comment:
Why is this the case? Why not pass the treatment names too? Seems very weird behavior

Reply:
Fixed by setting `d_t=None` when summarizing over all treatments.